components/lib/calendar/Calendar.js

```diff
@@ -3685,6 +3685,25 @@ export const Calendar = React.memo(
             return null;
         };
 
+        const checkDateIsDisabled = (month, year) => {
```

Comment:
This looks great I think for consistency lets just rename `checkDateIsDisabled` to `isDateDisabled` since that seems to be a pattern used through this codebase.

Reply:
I wanted to do it before, but it was used

Reply:
Oh let me look.

Reply:
Maybe it can be called isYearDisabled and isMonthDisabled like ng

Reply:
OK I just refactored `isDayDisabled` on your branch to be one single method.  I found it confusing that it was two differnet methods for isDateDisabled and isDayDisabled.  but i think the good name for it should be `isMonthYearDisabled`

Reply:
Okay,I'll go home later and update the code